include/freetds/odbc.h

```diff
@@ -152,7 +152,7 @@ struct _dheader
 {
 	SQLSMALLINT sql_desc_alloc_type;
 	/* TODO SQLLEN ?? see http://support.microsoft.com/default.aspx?scid=kb;en-us;298678 */
-	SQLSMALLINT sql_desc_count;
+	SQLUSMALLINT sql_desc_count;
```

Comment:
In this structure the types matches the relative type in descriptors, in this case `SQL_DESC_COUNT` from `SQLSetDescField`.

Reply:
Good point.  As it stands, the type doesn't match `desc_alloc_records`'s `count` argument, but I suppose that can and should change instead.